agent/src/flow_generator/flow_map.rs

```diff
@@ -472,20 +499,17 @@ impl FlowMap {
     }
 
     fn update_tcp_flow(&mut self, meta_packet: &mut MetaPacket, node: &mut FlowNode) -> bool {
+        let direction = meta_packet.direction;
+        let pkt_tcp_flags = meta_packet.tcp_data.flags;
+        node.tagged_flow.flow.flow_metrics_peers[direction as usize].tcp_flags |= pkt_tcp_flags;
+        self.update_flow(node, meta_packet);
         if node.tagged_flow.flow.signal_source == SignalSource::EBPF {
             // For eBPF data, since we don't know when the Flow will terminate at present, the
             // fixed timeout parameter is used uniformly, and the Flow will not be closed due to
             // the arrival of the packet.
-            // FIXME: We should check whether there are any remaining requests to be aggregated and
-            // close the flow as soon as possible.
             return false;
```

Comment:
In addition, because eBPF data does not have L4 information, the remaining steps of direction correction, state machine maintenance, SEQ acquisition, etc., do not need to be performed.

Reply:
已修改